cmd/changelog/main.go

```diff
@@ -17,11 +17,13 @@ import (
 
 // norevive
 const (
-	BUGFIX      = "Bugfix"
-	ENHANCEMENT = "Enhancement"
-	OTHER       = "Others"
-	BACKPORTING = "Backporting"
-	MISC        = "Misc"
+	BUGFIX          = "Bugfix"
+	DATAENHANCEMENT = "Data Enhancement"
+	ENHANCEMENT     = "Enhancement"
+	FEATURE         = "Feature"
+	OTHER           = "Others"
+	BACKPORTING     = "Backporting"
```

Comment:
I am just wondering if we need this item at all.....because it doesn't seem to be used in the CHANGELOG, right?